### ISSUE TITLE
Cura 5.6 -Won't Launch

### ISSUE THREAD

**Issue by jtprouty** [Ultimaker/Cura#17656]
### Cura Version

5.6.0

### Operating System

Window 11

### Printer

Kobra Max

### Reproduction steps

I installed 5.6.0 yesterday and it just brings up a blank screen when it launches. It also won't launch the previous version I had installed (4.12.0). I've uninstalled and reinstalled 5.6.0 several times with the same result.

### Actual results

No change

### Expected results

Launching the program.

### Add your .zip and screenshots here ⬇️

![Cura](https://github.com/Ultimaker/Cura/assets/153941398/a1f61bb7-f18b-4ece-b483-fe91badd9208)

**Comment by ranran2808** [comment]
I’m having the same issues, I’ve looked everywhere for results to fix this and I can’t find any, it is immensely frustrating

**Comment by jtprouty** [comment]
Uninstalled 5.6.0 and installed 5.5.0 - same results.

**Comment by ProfPun** [comment]
Have 
you tried running it as administrator?

**Comment by jtprouty** [comment]
That worked but I shouldn't have to run it as administrator.
s

**Comment by ProfPun** [comment]
Agreed, but it was the only thing I found to work. Sometimes , after running it once this way, then it would then open normally. Not sure if it is a Cura problem, Windows 10 or some sort of security issue? I am running Kaspersky.
